index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="#">Joaquin Calle Aranzolo</a>
+            <i class="fa fa-bars"></i>
+        </nav>
+    </header>
+    <main>
+        <h1>Hey there. I'm Joaquin.</h1>
+        <h3>I'm a Software Developer</h3>
```

Comment:
- [ ] Please double check the comment of the previous reviewer ...

- This should be like 
 ```html 
<h1>Hey there. I'm Joaquin</h1>
 <h2> I'm a Software Developer <.h2> 
```

- It means to make your `<h3></h3> ` an `<h2></h2>`